common/religion/religions/air_group.txt

```diff
@@ -46,8 +46,8 @@
 
 	# Placeholder
 	traits = {
-		virtues = { fickle zealous temperate }
-		sins = { stubborn cynical gluttonous }
+		virtues = { deceitful compassionate temperate }
```

Comment:
Why compassionate is a virtue?

Reply:
I've to much seen Avatar sorry

Reply:
Do I keep it ?

Reply:
Don't think so. Air praises madness so I suggest...
I would hit at least 3 personal traits.
I suggest:
`deceitful`, `arbitrary`, `fickle`, `lunatic` for virtues.
`honest`, `just`, `stubborn` for sins.

Reply:
Ok thank you !